### ISSUE TITLE
Failed to link to Python 2.7 on Fedora 23

### ISSUE THREAD

**Issue by cbisnett** [Vector35/binaryninja-api#464]
On Fedora 23 and some other distributions like Arch the Python 2.7 library is named `libpython2.7.so.1.0`. The `libpythonplugin.so` plugin fails to find the installed Python library.

This can be fixed by adding a symlink in the plugins directory to point to the system Python using a command like this: `ln -s <path to system python> plugins/libpython2.7.so.1` from the Binary Ninja root directory. On Fedora 23 the command is: `ln -s /lib64/libpython2.7.so.1.0 plugins/libpython2.7.so.1`

Output from `strace` shows the issue:

```
open("/home/chris/binary_ninja/plugins/libpython2.7.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/home/chris/binary_ninja/libpython2.7.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/etc/ld.so.cache", O_RDONLY|O_CLOEXEC) = 14
fstat(14, {st_mode=S_IFREG|0644, st_size=198067, ...}) = 0
mmap(NULL, 198067, PROT_READ, MAP_PRIVATE, 14, 0) = 0x7f11900fb000
close(14)                               = 0
open("/lib64/tls/libpython2.7.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib64/libpython2.7.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/lib64/tls/libpython2.7.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/lib64/libpython2.7.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
```

**Comment by jaredestroud** [comment]
This also works on Fedora 26.

**Comment by brianSchanbacher** [comment]
Fix for Fedora 27 -  Binary Ninja Version 1.1.1038 

`ln -s /lib64/libpython2.7.so.1.0 ./libpython2.7.so.1 `

This assumes your current working directory is the `binaryninja` folder.

**Comment by rssor** [comment]
Added as an option in settings.json as of build 1.1.1220-dev.

As an example, to have the UI use homebrew Python on macOS:

```
    "python" :
    {
        "interpreter": "/usr/local/Frameworks/Python.framework/Versions/2.7/lib/libpython2.7.dylib"
    }
```
